examples/web_agent/connect_database.py

```diff
@@ -0,0 +1,69 @@
+from dotenv import load_dotenv
+
+import pandas as pd
```

Comment:
Seeing this warning:
![image](https://github.com/user-attachments/assets/3d4616ed-abac-447a-8987-45be72984bc4)

Are we missing a step in the setup README?